### ISSUE TITLE
opencv: build failure if Eigen is not installed in osx

### ISSUE THREAD

**Issue by r9y9** [openframeworks/openFrameworks#4578]
In scripts/apothecary:

```
$ apothecary update ofxOpenCv
...

[ 93%] Building CXX object modules/contrib/CMakeFiles/opencv_contrib.dir/src/retinafilter.cpp.o
[ 93%] Building CXX object modules/contrib/CMakeFiles/opencv_contrib.dir/src/rgbdodometry.cpp.o
/Users/ryuyamamoto/cv/openFrameworks/scripts/apothecary/build/opencv/modules/contrib/src/rgbdodometry.cpp:65:12: fatal error: 'unsupported/Eigen/MatrixFunctions' file not found
#  include <unsupported/Eigen/MatrixFunctions>
           ^
1 error generated.
make[2]: *** [modules/contrib/CMakeFiles/opencv_contrib.dir/src/rgbdodometry.cpp.o] Error 1
make[1]: *** [modules/contrib/CMakeFiles/opencv_contrib.dir/all] Error 2
make: *** [all] Error 2
```

I'm definitely sure that adding `-DWITH_EIGEN=OFF` to the cmake fixes this problem. I can create a PR if you are ok to the change.

**Comment by danoli3** [comment]
This should already be no.

--   Other third-party libraries:
--     Use IPP:                     NO
--     Use Eigen:                   NO

Hmmm did you build with:
```./apothecary -t osx update ofxOpencv```

**Comment by danoli3** [comment]
Ah I have already made these changes in my latest branch.. I'll cherry pick all the script changes.

**Comment by r9y9** [comment]
```
--   Other third-party libraries:
--     Use IPP:                     NO
--     Use Eigen:                   YES (ver ..)
--     Use TBB:                     NO
```

Still happens. Will fixed by #4608
